fairlearn/metrics/_disparities.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Metrics for measuring disparity."""
+
+from ._metrics_engine import (
+    selection_rate_difference,
+    selection_rate_ratio,
+    true_positive_rate_difference,
+    true_positive_rate_ratio,
+    false_positive_rate_difference,
+    false_positive_rate_ratio)
+
+
+def demographic_parity_difference(y_true, y_pred, *, sensitive_features, sample_weight=None):
```

Comment:
Do we want to keep these `*` in the API? I'm sure I had a good reason for putting them there in the first place, but I can't quite remember it right now.

Reply:
It means keyword-name is required, so you can't write
`func(a, b, c)`
but instead you have to write
`func(a, b, c=c)`
We all agreed that's a good idea months ago when the project started, at least for sensitive_features since that's the part that's really different from other libraries like sklearn

Reply:
I knew I'd had a reason ;-)